New Gas Miner, the HES-Phaeron, Heavy Duty Gas Tanker

Rubeebeebee opened the pull request:
(I LOVE GAS MINING I LOVE GAS MINING)

I want to add another gas mining ship- so here is the HES-Phaeron, a tanker ship with a huge tank to carry lots of harvested gasses. It's niche is to simply harvest raw gasses and carry as much as it can to sell. It's automatically set to filter plasma into its tanks and flush the other gasses away into space. 

Highly modifiable, with plenty of space in and out of the ship to suit any end user's needs. 

(ALSO THIS IS MY FIRST EXPERIENCE CODING PLS BE GENTLE WITH ME)

<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
i added the HES-Phaeron, a medium sized gas miner ship equipped with a franky oversized gas tank and 2 mining pulsars to make the process go faster. 

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
We get a new miner in the shipyard filling it's own niche- with the Akasono being the entry level gas miner ship, and the Arribane being the all-around best atmos ship available, i wanted to make the Phaeron so it can fill a niche of just harvesting gasses and storing them en-masse. The arribane can do this, but they have dedicated filtered tanks, as well as a burn chamber. The Phaeron only has the one huge tank (which is larger than some cargo bays) with no other features (unless modified by the end user.) 

As is, the Phaeron can filter it's chosen gas into the tank, and vent the rest into space. The end user can modify this system to by the filter entirely, or modify it to their hearts content. 

In the end, the Phaeron's niche is that it can carry alot of gas all at once, and so it's worth selling then even if you don't have the knowledge, time, or patience to process said gasses into tritium or frezon.  


## How to test
<!-- Describe the way it can be tested -->
seems to be working just fine, try loading it up on a grid using the yml file. 

Ive tested it out earlier putting it into a shipyard, works just fine for me. 

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

<img width="548" height="842" alt="newphaeron live" src="https://github.com/user-attachments/assets/9a8d97fd-5999-438f-b06b-4a5c746324b2" />
<img width="545" height="845" alt="newphaeron" src="https://github.com/user-attachments/assets/8dfe853b-335d-4232-b1ae-540243dd22af" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
- [X] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: Added the HES Phaeron, a heavy-duty gas tanker.

Rubeebeebee committed:
I want to add another gas mining ship- so here is the HES-Phaeron, a tanker ship with a huge tank, but you'll have to carry either one filtered gas, or unfiltered gas.

monolith8319 committed:
chore: automatically update REUSE headers

sleepyyapril commented on Resources/Prototypes/_Mono/Shipyard/phaeron.yml:
```diff
@@ -0,0 +1,52 @@
+  id: Phaeron
+  parent: BaseVessel
+  name: HES Phaeron
+  description: Horizon Energy light-class gas miner- It's gigantic gas tanker is able to harvest and store a frightening amount of gas. Perfect for plasma harvesting in the Colossus sector.
```
```suggestion
  description: Part of Horizon Energy's latest gas mining initiative, this gigantic gas tanker is able to harvest and store a frightening amount of gas. Perfect for plasma harvesting in the Colossus sector.
```

Rubeebeebee replied:
thank you for helping me out on this!

sleepyyapril reviewed (commented):
grammar...

Rubeebeebee committed:
Update Resources/Prototypes/_Mono/Shipyard/phaeron.yml

Co-authored-by: sleepyyapril <123355664+sleepyyapril@users.noreply.github.com>

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
oops! i forgot to add a warp point + latejoint spawner gonna edit that

Rubeebeebee committed:
updating the phaeron now with warppoints and latejoin spawns

Rubeebeebee committed:
Merge branch 'Phaeron-branch' of https://github.com/Rubeebeebee/Monolith into Phaeron-branch

Rubeebeebee committed:
added decals to the phaeron

Merge branch 'Phaeron-branch' of https://github.com/Rubeebeebee/Monolith into Phaeron-branch

Rubeebeebee commented on Resources/Prototypes/_Mono/Shipyard/phaeron.yml:
```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Author Info
+# GitHub: starch70
```
i gotta change these to my name

Rubeebeebee replied:
nevermind i cant change that from the template i used

Rubeebeebee committed:
Merge branch 'Phaeron-branch' of https://github.com/Rubeebeebee/Monolith into Phaeron-branch

monolith8319 committed:
chore: automatically update REUSE headers

Redrover1760 committed:
Merge remote-tracking branch 'upstream/main' into pr/Rubeebeebee/1401

Redrover1760 commented on Resources/Prototypes/_Mono/Shipyard/phaeron.yml:
```diff
@@ -0,0 +1,54 @@
+      stationProto: StandardFrontierVessel
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: 'Phaeron HES{1}'
```
Should be CIV

Rubeebeebee replied:
> Should be CIV

gotcha, will update it

Rubeebeebee replied:
> Should be CIV

there we go! updated it!

Rubeebeebee committed:
updated the .yml so it says mapNameTemplate: 'Phaeron CIV{1}' instead of mapNameTemplate: 'Phaeron HES1}'

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into Phaeron-branch

Redrover1760 reviewed (changes requested):
If you could trace the green pipe in catwalk to make it clear the main external gaslock is linked would be good. Its a bit unclear if you don't have a t-ray what the manual valve is linked to

the atmos tech locker doesn't use the hardsuit variant for the atmos modsuit, not sure if intentional

if you swapped the pacman with the smes you can avoid having to put the critical HV line to the substation under walls and instead under catwalk so its visible

machinery, especially in engine room could use decals to divide it from the catwalk. Of particular note is the grav gen and gyroscope, smes and pacman

lacks a firelock under the maint door leading to the engine room. Check your other ship for missing firelocks too, I forgot to.

overall very good looking ship, creative concept and atmos work as usual from ya :>

Redrover1760 commented:
missing a mini-pacman + welding fuel to power the gas mining drills

Rubeebeebee commented:
> missing a mini-pacman + welding fuel to power the gas mining drills

we do have a mini pacman! check the the spot by docking bay, next to the orebox. Although i did needed to add  fuel for them! thanks! ill get on that

Rubeebeebee committed:
added a welding fuel wall locker next to the JPAC generator.

Rubeebeebee commented:
> missing a mini-pacman + welding fuel to power the gas mining drills

i have now added a welding fuel wall locker next to the JPAC generator!

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee committed:
fixed all the stuff mentioned in the requests.

added a atmosphere network console in place of the atmospheric alerts console, added gas pipe sensors on every pipe network of note.

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
> If you could trace the green pipe in catwalk to make it clear the main external gaslock is linked would be good. Its a bit unclear if you don't have a t-ray what the manual valve is linked to
> 
> the atmos tech locker doesn't use the hardsuit variant for the atmos modsuit, not sure if intentional
> 
> if you swapped the pacman with the smes you can avoid having to put the critical HV line to the substation under walls and instead under catwalk so its visible
> 
> machinery, especially in engine room could use decals to divide it from the catwalk. Of particular note is the grav gen and gyroscope, smes and pacman
> 
> lacks a firelock under the maint door leading to the engine room. Check your other ship for missing firelocks too, I forgot to.
> 
> overall very good looking ship, creative concept and atmos work as usual from ya :>

good news! fixed all of the issues you mentioned here, plus more!

I added a atmospheric network console in place of the atmospheric alerts console, plus gas sensors on every gas network of note. 
<img width="552" height="870" alt="newerphaeron live" src="https://github.com/user-attachments/assets/e524c8b0-09bf-4a5d-8a5c-838a1efc0e18" />
<img width="1120" height="687" alt="newer phaeron engine room" src="https://github.com/user-attachments/assets/f1e9024a-6037-42b4-b226-dd3c276c0aeb" />
<img width="793" height="613" alt="phaeron pipes" src="https://github.com/user-attachments/assets/53815194-b715-4237-91b2-d54a8c7df761" />
<img width="1182" height="779" alt="atmospheric monitor" src="https://github.com/user-attachments/assets/5ba1b155-d781-499d-aee0-15783b54be6e" />

Rubeebeebee commented:
> If you could trace the green pipe in catwalk to make it clear the main external gaslock is linked would be good. Its a bit unclear if you don't have a t-ray what the manual valve is linked to
> 
> the atmos tech locker doesn't use the hardsuit variant for the atmos modsuit, not sure if intentional
> 
> if you swapped the pacman with the smes you can avoid having to put the critical HV line to the substation under walls and instead under catwalk so its visible
> 
> machinery, especially in engine room could use decals to divide it from the catwalk. Of particular note is the grav gen and gyroscope, smes and pacman
> 
> lacks a firelock under the maint door leading to the engine room. Check your other ship for missing firelocks too, I forgot to.
> 
> overall very good looking ship, creative concept and atmos work as usual from ya :>

ive also added a small walkway next to the gas nozzle. 

also, the atmos locker is fine without a hardsuit, the firesuit is space worthy. the atmos tech EVA wall locker is fine being a softsuit.

Rubeebeebee commented:
WOOOO YEAH BABEEEY!! THANK YOU @Redrover1760 !!!!
